Alignment/MuonAlignmentAlgorithms/python/gather_cfg.py

```diff
@@ -102,6 +102,30 @@
 
 process = cms.Process("GATHER")
 
+process.load("Geometry.MuonNumbering.muonNumberingInitialization_cfi")
+process.load("Geometry.DTGeometry.dtGeometry_cfi")
+process.load("Geometry.RPCGeometry.rpcGeometry_cfi")
+process.load("Geometry.CSCGeometry.cscGeometry_cfi")
+process.load("Geometry.CommonDetUnit.bareGlobalTrackingGeometry_cfi")
+
+#add TrackDetectorAssociator lookup maps to the EventSetup
+process.load("TrackingTools.TrackAssociator.DetIdAssociatorESProducer_cff") 
+from TrackingTools.TrackAssociator.DetIdAssociatorESProducer_cff import *  
+from TrackingTools.TrackAssociator.default_cfi import *       
+
+
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+
+process.MuonNumberingInitialization = cms.ESProducer("MuonNumberingInitialization")
+process.MuonNumberingRecord = cms.ESSource( "EmptyESSource",
+    recordName = cms.string( "MuonNumberingRecord" ),
+    iovIsRunNotTime = cms.bool( True ),
+    firstValid = cms.vuint32( 1 )
+)
+
+process.load("Configuration.StandardSequences.GeometryDB_cff")
+process.load('Configuration.StandardSequences.MagneticField_38T_PostLS1_cff')
```

Comment:
@mmusich thanks for suggestion. The content of "MagneticField_AutoFromDBCurrent_cff" says that it is obsolete and will be deprecated at some point, and suggests ti replace it with "MagneticField_cff".